### PR TITLE
Issue316/Contact Form

### PR DESCRIPTION
**Pull request by George-Onyango**
# Description

This pull request create a new contact form that a user is able to fill in order to get into contact with customers that might have reviews on the organization

Fixes # (issue) - (#316 )

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

# Screenshot (optional)

# Web

Screenshot from 2023-03-03 15-53-28

# Mobile

Screenshot from 2023-03-03 15-53-36

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 
Please also list any relevant details for your test configuration if necessary.
- [x] Test responsiveness on design change

# Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by George-Onyango** [comment]
@IanOdhiambo9  requesting review

File: libs/pages/elewa/contact/src/lib/components/contact-banner/contact-banner.component.html [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+  <div class="contact-info">
+    <div id="questions">
+      <h2 class="sudHeading">Questions</h2>
+      <p class="text">{{questionEmail}}</p>
+    </div>
+
+    <div id="press">
+      <h2 class="sudHeading">Press</h2>
+      <p class="text">{{pressEmail}}</p>
```
Comment (IanOdhiambo9) [inline libs/pages/elewa/contact/src/lib/components/contact-banner/contact-banner.component.html:12]: This was not part of this issue

**Review by IanOdhiambo9 — CHANGES_REQUESTED** [review]

**Comment by IanOdhiambo9** [comment]
The icon on the dropdown is not vertically centered on the field

**Comment by IanOdhiambo9** [comment]
This issue has been completed

**Pull request closed without merging**